### DISCUSSION TITLE
How to supress unwanted GPL ICMP_INFO PING

### DISCUSSION THREAD

**Discussion by Sazerak** [Security-Onion-Solutions/securityonion#7196] [Unsupported Versions] [answered]
Can someone tell me if there's a way to add granularity to the GPL ICMP_INFO PING rules? Specifically, I'm getting thousands of NIX and BSDtype rule alerts when my IoT devices ping the IoT gateway. I seem to remember reading (maybe somewhere here) that these rules were set to any --> home_net instead of external_net --> home_net in order to detect lateral movement, but maybe I'm remembering wrong because mine are set to external_net --> home_net and yet they still alert when they shouldn't (or at least I don't think they should).

How would I go about properly mitigating this? It really seem to be strictly related to my IoT VLAN (or maybe any VLAN other than the primary as I only have the primary and the IoT). Basically, I don't want to disable the rule, but I'd like it to not alert when devices ping the IoT gateway, similarly to how I'm sure I must have lots of devices pinging my primary gateway but that isn't bringing up any alerts.

Any guidance is appreciated.

**Comment by dougburks** [comment, marked as answer]
> I seem to remember reading (maybe somewhere here) that these rules were set to any --> home_net instead of external_net --> home_net in order to detect lateral movement, but maybe I'm remembering wrong because mine are set to external_net --> home_net and yet they still alert when they shouldn't (or at least I don't think they should).

EXTERNAL_NET is set to `any` by default to detect lateral movement:
https://docs.securityonion.net/en/2.3/suricata.html?highlight=external_net#configuration

You could suppress the alerts for certain addresses:
https://docs.securityonion.net/en/2.3/managing-alerts.html

or use BPF to filter out the traffic altogether:
https://docs.securityonion.net/en/2.3/bpf.html
